matrix_client/api.py

```diff
@@ -231,6 +231,18 @@ def send_emote(self, room_id, text_content):
             self.get_emote_body(text_content)
         )
 
+    def send_notice(self, room_id, text_content):
```

Comment:
I'm all for convenience methods but I'd prefer it if the API kept to strictly wrapping the Client Server API or some other process background process.